### DISCUSSION TITLE
one time donation

### DISCUSSION THREAD

**Discussion by lfg6000** [dtcooper/raspotify#508] [Q&A] [answered]
Jason,

First of all thank you for taking over the lead on raspotify.  The raspotify service is major feature in my home brewed voice controlled home automation system.

Just upgraded from .31.3 to .31.8 and found your comments in the new conf file quite helpful.

I would happy to make a one time donation.  Is it possible to make a one time donation? (does the github sponsor feature offer a one time donation button)

Thanks,
Louie

**Comment by JasonLG1979** [comment, marked as answer]
1st thanks. But to answer your question, I'm not sure? I'd imagine you could just sponsor me and then cancel before the next month?

**Reply by lfg6000** [reply]
okay i will do that.
by the way i created a website you might find useful
SpotifyFinder.com
 - Use this website to search for a song/artist/album across your Spotify playlists.
 - Use this website to find duplicates across playlists or in the same playlists and remove them.

**Reply by JasonLG1979** [reply]
Well I look at my sponsor dashboard for the 1st time in forever and it turns out you can enable one time donations so I've done that.
